### ISSUE TITLE
In the top of the list should be the tasks from the current day

### ISSUE THREAD

**Issue by georgianacretu** [paulbrie/owntribe#11]

**Comment by paulbrie** [comment]
The tasks already appear by creation order (most recent first). Would you like to see something more specific?

**Comment by septimiucernat** [comment]
Yes, order the tasks by their date. Ex: I have added 3 tasks, two for today and one for tomorrow. I want to see this details

Today
Task 1 ----------------
Task 2 ----------------

Tomorrow
Task 3 ----------------

And if the task is not finished but the day passes, show

Yesterday
Task 1 ----------------
Task 2 ----------------

Today
Task 3 ----------------
